libs/libc/string/lib_memmem.c

```diff
@@ -42,6 +42,7 @@
  *
  ****************************************************************************/
 
+#undef memmem /* See mm/README.txt */
```

Comment:
@xiaoxiang781216 in general I see the `#undef xxxx /* See mm/README.txt */` used in `string.h` APIs, but the content of `mm/README.txt` has description only for APIs from `stdlib.h`. I'm not sure if the same is applied to `string.h` APIs so maybe `libs/libc/string` should be cleaned up instead of adding undefs

Reply:
Yes, the undef is needed for string too.

Reply:
Should we update `mm/README.md` to mention string.h APIs as well?

Reply:
could be.